code/_helpers/_global_objects.dm

```diff
@@ -0,0 +1,8 @@
+var/datum/gear_tweak/color/gear_tweak_free_color_choice_
+/proc/gear_tweak_free_color_choice()
+	if(!gear_tweak_free_color_choice_) gear_tweak_free_color_choice_ = new()
```

Comment:
That should be splitted in two lines.

Reply:
fixed

Reply:
Fixed?

Reply:
yes

Reply:
first time i misunderstood but should be fixed now